integration-cli/docker_cli_volume_test.go

```diff
@@ -299,3 +299,15 @@ func (s *DockerSuite) TestVolumeCliCreateLabelMultiple(c *check.C) {
 		c.Assert(strings.TrimSpace(out), check.Equals, v)
 	}
 }
+
+func (s *DockerSuite) TestVolumeCliRmForce(c *check.C) {
```

Comment:
Maybe in a separate test (as it requires `SameHostDeamon`) you can create a volume, manually remove it from `<docker root>/volumes/<id>`, then `docker volume rm -f` it.